app/shared/models.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import unicode_literals
+
+from django.db import models
+
+class CommonInfo(models.Model):
+    date_created = models.DateTimeField(auto_now=False, auto_now_add=True)
```

Comment:
By default `auto_now` and `auto_now_add` are False so we don't need to explicitly state the boolean value `False` for them
Refer to https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.DateTimeField